test/unit/uk/gov/hmrc/exports/config/AppConfigSpec.scala

```diff
@@ -16,125 +16,86 @@
 
 package uk.gov.hmrc.exports.config
 
-import java.util.UUID
-
-import com.typesafe.config.{Config, ConfigFactory}
-import play.api.{Configuration, Environment}
-import uk.gov.hmrc.exports.base.UnitSpec
-import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
-
-class AppConfigSpec extends UnitSpec {
-
-  private val validAppConfig: Config =
-    ConfigFactory.parseString("""
-      |urls.login="http://localhost:9949/auth-login-stub/gg-sign-in"
-      |mongodb.uri="mongodb://localhost:27017/customs-declare-exports"
-      |microservice.services.auth.host=localhostauth
-      |microservice.services.auth.port=9988
-      |microservice.services.customs-declarations.host=remotedec-api
-      |microservice.services.customs-declarations.port=6000
-      |microservice.services.customs-declarations.api-version=1.0
-      |microservice.services.customs-declarations.submit-uri=/declarations
-      |microservice.services.customs-declarations.cancel-uri=/declarations/cancel
-      |microservice.services.customs-declarations.bearer-token=Bearer DummyBearerToken
-      |microservice.services.customs-declarations-information.host=localhostd
-      |microservice.services.customs-declarations-information.port=9834
-      |microservice.services.customs-declarations-information.api-version=1.0
-      |microservice.services.customs-declarations-information.bearer-token=Bearer cdi-bearer-token
-      |microservice.services.customs-declarations-information.client-id=cdi-client-id
-      |microservice.services.customs-declarations-information.fetch-mrn-status=/mrn/ID/status
-      |microservice.services.customs-data-store.host=localhost
-      |microservice.services.customs-data-store.port=6790
-      |microservice.services.customs-data-store.verified-email-path=/customs-data-store/eori/EORI/verified-email
-    """.stripMargin)
```

Comment:
It looks like you've changed the purpose of this test class to now test the actual values in application.conf (rather than as before it was just testing the functionality of AppConfig in retrieving values or throwing exceptions). This means changing a value in application.conf will now also need to be reflected in this test also.

I agree though in that I don't see the point in these tests as they are mostly exercising the Play Bootstrap library rather than our own code. :thinking:

Reply:
> This means changing a value in application.conf will now also need to be reflected in this test also.
I think this is a good thing :smiley: and a more effective way to test the values loaded by AppConfig are there and the expected ones.